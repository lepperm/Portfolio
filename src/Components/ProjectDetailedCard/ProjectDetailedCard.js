/**
 * @fileoverview ProjectDetailedCard component.
 */

import React from "react";
import PropTypes from "prop-types";

/**
 * ProjectDetailedCard component
 *
 * @return {Component} ProjectDetailedCard component
 */
export const ProjectDetailedCard = () => {
  return <div>ProjectDetailedCard</div>;
};

/**
 * Prop types for ProjectDetailedCard component
 */
ProjectDetailedCard.propTypes = {
  //
};
