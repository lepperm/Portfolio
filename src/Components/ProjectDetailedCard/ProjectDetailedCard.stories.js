/**
 * @fileoverview Story for ProjectDetailedCard component.
 */

import React from "react";
import { ProjectDetailedCard } from "./ProjectDetailedCard";

export default {
  title: "ProjectDetailedCard",
  component: ProjectDetailedCard,
};

export const Basic = () => <ProjectDetailedCard />;
