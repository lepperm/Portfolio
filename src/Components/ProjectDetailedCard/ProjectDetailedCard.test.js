/**
 * @fileoverview Tests for ProjectDetailedCard component.
 */

import React from "react";
import { shallow } from "enzyme";
import { ProjectDetailedCard } from "./ProjectDetailedCard";

describe("ProjectDetailedCard", () => {
  test("matches snapshot", () => {
    const wrapper = shallow(<ProjectDetailedCard />);
    expect(wrapper).toMatchSnapshot();
  });
});
