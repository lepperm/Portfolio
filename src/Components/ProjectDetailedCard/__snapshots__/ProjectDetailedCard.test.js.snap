// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectDetailedCard matches snapshot 1`] = `ShallowWrapper {}`;
